### DISCUSSION TITLE
Is it possible to use encrypted EBS-Volumes and encrypted AMIs?

### DISCUSSION THREAD

**Discussion by mariusfilipowski** [runs-on/runs-on#109] [Q&A]
Since this is considered a best practice, is this possible? Even with an own key?
How much longer will the startup of an instance take?

**Comment by crohr** [comment]
I haven't tested enabling EBS encryption, but can definitely investigate. I don't think it would alter the boot time much, but I may be wrong.

**Comment by crohr** [comment]
Forgot to update this discussion, but EBS encryption is available since quite a while. It does make the boot time a bit more erratic (adds ~10-20s in my tests, but depends on your AMI).
